libs/core/tests/unit_tests/test_tools.py

```diff
@@ -987,3 +988,33 @@ def foo(
         },
         "required": ["bar", "baz"],
     }
+
+
+def test_convert_from_runnable() -> None:
+    def f(x: List[int]) -> int:
```

Comment:
Is there a version of this that uses a single `RunnableLambda`?

Reply:
added